torch/_inductor/ir.py

```diff
@@ -6240,12 +6245,7 @@ def create(
         algorithm,
     ):
         transposed = True
-        (
-            inputs,
-            constant_args,
-            kernel_layout,
-            _,
-        ) = _prepare_convolution_fusion_create(
+        (inputs, constant_args, kernel_layout, _,) = _prepare_convolution_fusion_create(
```

Comment:
nit: formatting change

Reply:
I think lintrunner complained if I didn't have this change 😅